medical_family/__openerp__.py

```diff
@@ -42,7 +42,7 @@
 {
 
     'name': 'Medical Family',
-    'version': '8.0.1.1.0',
+    'version': '9.0.1.1.0',
```

Comment:
We restart our internal semantic versioning on migration, so this would be `9.0.1.0.0`